.travis.yml

```diff
@@ -30,7 +32,7 @@ install:
 
 script:
   - composer check-code
-  - if [ "$TRAVIS_PHP_VERSION" == "7.1" ]; then composer test-cov; else composer test; fi
```

Comment:
You can leave it fixed at `== "8.0"` here and below 👍 

(not sure if it even makes sense to send multiple coverage reports)